src/sys/unix.rs

```diff
@@ -79,6 +79,38 @@ macro_rules! syscall {
     }};
 }
 
+#[cfg(any(
+    target_os = "dragonfly",
+    target_os = "freebsd",
+    target_os = "ios",
+    target_os = "macos",
+    target_os = "netbsd",
+    target_os = "openbsd",
+))]
+const fn max_iov() -> usize {
```

Comment:
Might as well make these constants. I was too lazy to do that in libstd when I added `IOV_MAX` and `UIO_MAXIOV` to libc.

Reply:
Done :)